chatreactions/leaderboards.md

```diff
@@ -1,22 +1,47 @@
+# What are chat reactions?:
+
+Chat reactions are events in which multiple individuals compete to type a word the fastest. They can spawn randomly from the bot or be initiated as a duel (`$cr duel`).
+
+## Example of a chat reaction:
+
+chat reaction
+
+type: `sailing`
+
+**winners**
+
+🥇Ayestk in `1.89s`
+
+🥈Scarlett in `2.73s`
+
+🥉M.axz in `3.42s`
+
```

Comment:
maybe change this to a picture, put pictures in the assets folder.

Reply:
why lower case?